prisma/coordenadores.csv

```diff
@@ -0,0 +1,3 @@
+type;name;email;password;updated_at
+Coordenador;Alexandre Passito de Queiroz;passito@icomp.ufam.edu.br;$2b$10$eZ2mRt5QSbx4qTPJCGOOIuEcre6MXD9bJorOElHfJ7.iPrdiOBOa.;'2023-05-19'
+Coordenador;Ana Lucia Machado dos Santos;analuciamachado@icomp.ufam.edu.br;$2b$10$5sIdlIY25A61FPHMxtjkweX6vJhc/soxrQfnPgDH/K13u14pzwXSi;'2023-05-19'
```

Comment:
Remove esse arquivo daqui, amg. Não to achando muito segura deixar no github o login dos professores.

Reply:
Quando eu for rodar em produção eu coloco esses arquivos na pasta